notify/src/inotify.rs

```diff
@@ -35,7 +37,7 @@ struct EventLoop {
     event_loop_rx: Receiver<EventLoopMsg>,
     inotify: Option<Inotify>,
     event_handler: Box<dyn EventHandler>,
-    watches: HashMap<PathBuf, (WatchDescriptor, WatchMask, bool)>,
+    watches: HashMap<PathBuf, (WatchDescriptor, WatchMask, bool, bool)>,
```

Comment:
```suggestion
    /// - watches HashMap Value tuple (WatchDescriptor, WatchMask, is_recursive, is_dir)
    watches: HashMap<PathBuf, (WatchDescriptor, WatchMask, bool, bool)>,
```

Reply:
done

Reply:
can this be a doc comment ? `///`

Reply:
thanks, but please also remove the upper one :)

Reply:
Done. I thought you meant the comment above.